moveit.rosinstall

```diff
@@ -28,8 +20,4 @@
     local-name: srdfdom
     uri: https://github.com/ros-planning/srdfdom.git
     version: melodic-devel
-- git:
-    local-name: moveit_tutorials
-    uri: https://github.com/ros-planning/moveit_tutorials.git
-    version: melodic-devel
```

Comment:
it sure would be nice if all these repos were merged together - we wouldn't have this issue